NanoVNASaver/Calibration.py

```diff
@@ -37,8 +37,7 @@ def __init__(self, app):
         from .NanoVNASaver import NanoVNASaver
 
         self.app: NanoVNASaver = app
-
-        self.setMinimumSize(450, 600)
+        self.setMinimumSize(450, 800)
```

Comment:
Can't have minimum height be more than 700-ish. I would prefer not increasing it at all unless strictly needed.

Reply:
To do so I will have to put css class to each component of the calibration window. Now it use the global values for those type of components.

Reply:
done